django_loci/static/django-loci/js/loci.js

```diff
@@ -343,6 +344,92 @@ django.jQuery(function ($) {
         };
     }
 
+    // returns placed marker
+    function get_marker() {
+        var map = getMap();
+        if (map !== undefined) {
+            var layer;
+            for (layer in map._layers) {
+                if (map._layers.hasOwnProperty(layer)) {
+                    if (map._layers[layer].hasOwnProperty('_latlng')) {
+                        if (map._layers[layer]._events.click === undefined) {
+                            return map._layers[layer];
+                        }
+                    }
+                }
+            }
+        } else {
+            window.setTimeout(get_marker, 100);
+        }
+    }
+
+    // returns map's feature group
+    function get_featureGroup() {
+        var map = getMap();
+        var layer;
+        for (layer in map._layers) {
+            if (map._layers.hasOwnProperty(layer)) {
+                if (map._layers[layer].hasOwnProperty('_layers')) {
+                    return map._layers[layer];
+                }
+            }
+        }
+    }
+
+    // set marker
+    function update_map(geocode_api) {
+        if ($addressInput.val() !== '') {
+            window.fetch(geocode_api + '?address=' + $addressInput.val())
+                    .then(function (resp) {
+                    var data_promise = resp.json();
+                    data_promise.then(function (data) {
+                        if (resp.status === 200) {
+                            var current_marker_lat = get_marker() !== undefined ?
+                                                        get_marker().getLatLng().lat : null;
+                            var current_marker_lng = get_marker() !== undefined ?
+                                                        get_marker().getLatLng().lng : null;
+
+                            if (current_marker_lat !== data.lat || current_marker_lng !== data.lng) {
+                                var wkt = '{ "type": "Point", "coordinates": [ ' +
```

Comment:
why did you call this var `wkt`?
WKT is something like: `POINT (30 10)`. Maybe is better to call it geojson.